apps/src/redux/instructions.js

```diff
@@ -321,6 +327,7 @@ export const determineInstructionsConstants = config => {
     shortInstructions2,
     longInstructions,
     teacherMarkdown,
-    hasContainedLevels
+    hasContainedLevels,
+    levelVideos: videos,
```

Comment:
I might consistently just call this either videos or levelVideos rather than switching between the two.